### PR TITLE
afc: add `progress_bar` option when pulling files

### LINKED ISSUE #1300: [suggestion] AFC Pull "large" files trange to range
Due to this change:

https://github.com/doronz88/pymobiledevice3/blob/937b52d08930edc8bd82c1614ef663ab6b0677a1/pymobiledevice3/services/afc.py#L242-L244

when collecting multiple files, for example crash_logs, if I do:

```python
        with CrashReportsManager(device) as crash_manager:
            for item in tqdm(crash_manager.ls("/"), desc="Pulling crash logs", leave=False):
                  crash_manager.pull("my_path", item)
```

And if there are files larger than MAX_READ_SIZE, I end up having several progress bar, while I just want one, that will disappear and the "additional" ones remain.

```bash
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 5/5 [00:00<00:00,  7.84it/s]
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 5/5 [00:00<00:00,  8.12it/s]
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 5/5 [00:00<00:00,  7.86it/s]
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 5/5 [00:00<00:00,  8.06it/s]
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 5/5 [00:00<00:00,  8.13it/s]

MY PROGRESS BAR DISAPPEARED AS PER LEAVE=FALSE
```

Would @doronz88 consider __change trange by range__?

### PR DESCRIPTION
**Pull request by dario-br**
Resolves doronz88/pymobiledevice3#1300

### CODE REVIEW

**Comment by dario-br** [comment]
This pull requests does not change the CLI for AFC nor Developer, since those commands just pull one file at a time.

File: pymobiledevice3/services/afc.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 from pygments import formatters, highlight, lexers
 from pygnuutils.cli.ls import ls as ls_cli
 from pygnuutils.ls import Ls, LsStub
-from tqdm import trange
+from tqdm.auto import trange
```
Comment (doronz88) [inline pymobiledevice3/services/afc.py:25]: what is the reason for this diff?

**Review by doronz88 — CHANGES_REQUESTED** [review]
Thanks for the PR! Please just explain the use of `tqdm.auto` or restore the old import

File: pymobiledevice3/services/afc.py (hunk 1 above)
Comment (dario-br) [reply pymobiledevice3/services/afc.py:25]: In the case pymobiledevice3 is used outside CLI environment, ex. jupyter notebooks, tqdm may choose the best option possible. If you think it is not valuable, I revert the change, it is not a big thing.

Comment (doronz88) [reply pymobiledevice3/services/afc.py:25]: Nice :) Didn't know that

**Review by doronz88 — APPROVED** [review]

**Pull request merged**